style.css

```diff
@@ -0,0 +1,21 @@
+
+
+#title {color: blue;}
+#main{  background-color: gray;
+}
+#image {max-width:100%; height:auto;
```

Comment:
Please leave a blank line between closing bracket and new line of code. This is done to have a clean and easy to read code. It is common practice among programmers.